platform/platform-scheduler/src/main/java/ddf/platform/scheduler/CommandJob.java

```diff
@@ -49,13 +50,12 @@
 
     private static final Security SECURITY = Security.getInstance();
 
-    public Subject getSystemSubject() {
+    protected Subject getSystemSubject() {
```

Comment:
:question: Can this be private?

Reply:
This method is redefined in the unit tests.